mimic/test/test_maas.py

```diff
@@ -382,36 +331,42 @@ def test_agenthostinfo(self):
         """
         fetch agent host info
         """
-        req = request(self, self.root, "GET", self.uri + '/views/agent_host_info', '')
+        for q in range(4):
+            req = request(self, self.root, "GET",
+                          self.uri + '/views/agent_host_info?entityId='+self.entity_id)
+            resp = self.successResultOf(req)
+            self.assertEquals(resp.code, 400)
+            data = self.get_responsebody(resp)
+            self.assertEquals(True, 'Agent does not exist' in json.dumps(data))
+        req = request(self, self.root, "GET",
+                      self.uri + '/views/agent_host_info?entityId='+self.entity_id, '')
```

Comment:
nit: space around the operator. Also, why send the empty string for the headers?